### PR TITLE
LPS-194245 Adapt tests

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/import/ImportResults.tsx [hunk 1]
```diff
@@ -184,7 +169,7 @@ export function getResultsText(importResults: Results) {
 
 	Object.entries(importResults).forEach(([key, results]) => {
 		if (results.length) {
-			const {titles} = RESULTS_DATA[key as keyof ResultsData];
+			const {titles} = RESULTS_DATA[key as keyof typeof RESULTS_DATA];
```
Comment (victorg1991) [inline modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/import/ImportResults.tsx:179]: Divino

**Comment by ealonso** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by alessandromalves** [comment]
The failures are not related to this PR [(comparison with acceptance routine here)](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1692905243235&testrayRunIdA=2411794872&testrayRunIdB=2412170695). See https://github.com/brianchandotcom/liferay-portal/pull/139695

**Comment by alessandromalves** [comment]
ci:close

**Pull request closed without merging**